src/docfx/template/TemplateEngine.cs

```diff
@@ -204,6 +204,7 @@ private RenderType GetTocRenderType()
             }
 
             var jsonSchema = JsonUtility.DeserializeData<JsonSchema>(schemaString, new FilePath(schemaFilePath));
+            jsonSchema.SchemaType = mime.ToLowerInvariant();
```

Comment:
If `SchemaType` is a part of `JsonSchema` definition, that means people can override it in the schema definition. Is it intentional? If so, only assign the value if there is no existing `SchemaType`.

Reply:
OK,  have removed it and added fallback logic.